src/index.tsx

```diff
@@ -62,23 +76,36 @@ createRoot(container).render(
   <StrictMode>
     <Provider store={store}>
       <FeatureFlagsProvider>
-        <QueryClientProvider client={queryClient}>
-          <HashRouter>
-            <LanguageProvider>
-              <Web3Provider>
-                <ApolloProvider client={apolloClient}>
-                  <BlockNumberProvider>
-                    <Updaters />
-                    <ThemeProvider>
-                      <ThemedGlobalStyle />
-                      <App />
-                    </ThemeProvider>
-                  </BlockNumberProvider>
-                </ApolloProvider>
-              </Web3Provider>
-            </LanguageProvider>
-          </HashRouter>
-        </QueryClientProvider>
+        <StatsigProvider
+          user={{
+            userID: getDeviceId(),
+            customIDs: { session_id: String(getSessionId() ?? -1) },
+          }}
+          // TODO: replace with proxy and cycle key
+          sdkKey="client-1rY92WZGidd2hgW4x1lsZ7afqm1Qfr3sJfH3A5b8eJa"
```

Comment:
this should be stored in an `.env` variable, no?

Reply:
i'm not sure... i figured it's fine for our testrun experiment because we're already planning to cycle the API key when we start proxying. plus it's easier to setup.